File: dailyques/Q66.cpp
//Q66.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& a) {
        for(int i=a.size()-1;i>=0;i--){
            if(a[i]<9){ a[i]++; return a; }
            a[i]=0;
        }
        a.insert(a.begin(),1);
        return a;
    }
};

File: dailyques/Q66.py
#Q66.py
class Solution:
    def plusOne(self, a):
        for i in range(len(a)-1, -1, -1):
            if a[i] < 9:
                a[i] += 1
                return a
            a[i] = 0
        return [1] + a
